packages/mobile/package.json

```diff
@@ -192,6 +192,8 @@
     "escape-string-regexp": "^1.0.5",
     "eslint": "^6.8.0",
     "jest-fetch-mock": "^3.0.3",
+    "jest-html-reporter": "^3.3.0",
+    "jest-junit": "^12.0.0",
```

Comment:
Can we move these to the root `package.json` (`jest-junit` is already declared there)
so it can be used and updated for all packages?

This would prevent unnecessary duplication of different versions in `yarn.lock` too.

Reply:
- [x] Move the dependencies to the root package.json.
- [x] Add `index.html`, a human readable test report, to artifacts to give greater details on passed tests.

![Screen Shot 2021-04-09 at 11 07 39 AM](https://user-images.githubusercontent.com/26950305/114222786-dc11ef00-9923-11eb-845e-87b25a0a117e.png)

- [x] Confirm that Circle Ci Insights is tracking meta test history.

![Screen Shot 2021-04-09 at 11 02 07 AM](https://user-images.githubusercontent.com/26950305/114222182-18911b00-9923-11eb-9a36-bd9aebf7d55a.png)

- [x] Confirm that Failed tests are providing details in test results tab.

![Screen Shot 2021-04-09 at 10 55 40 AM](https://user-images.githubusercontent.com/26950305/114221839-a9b3c200-9922-11eb-8d3a-bbcaa13f7768.png)